pages/landing/migrate-from-heroku/pricing/index.html

```diff
@@ -0,0 +1,85 @@
+slug: index-page
+
+title: Pricing
+excerpt: Configure the plan just right for your team.
+permalink: /migrate-from-heroku/pricing/
+
+pricing:
+  - title: Free
+    description: For solopreneuers and personal projects.
+    price: Free
+    image: /assets/img/grunty-free@2x.png
+    image_styles: "max-width: 150px; margin-top: 18px"
+    features:
+      - name: 1 App
+        description: An app is a single service that you want to migrate from Heroku to AWS. This plan allows you to migrate & run only one app. For more apps, please see the other plans.
+      - name: Single AWS Account
+        description: You can only migrate to a single AWS account with this plan. For multi-account setup, please refer to the Enterprise plan.
+      - name: 1 User
+        description: A user is a human or machine that accesses the Gruntwork code repos, training materials, or support services. For more than 10 users, see Gruntwork Enterprise.
+      - name: Gruntwork Lite Stack
+        description: You get access to the Gruntwork Lite Stack, which includes modules for services such as ECS Fargate, ALB, ECR and Aurora Serverless.
+      - name: Access to Docs
+        description: This plan gets you access to all the modules documentation and any production guides for free. For access to the DevOps Training Library, see the Enterprise plan.
+      - name: $0 USD estimated AWS costs*
+        description: This plan is eligible for the AWS Free Tier. Please contact us for more information.
+
+  - title: Pro
+    description: For startups, small businesses planning on running a few apps.
+    price: "495"
+    image: /assets/img/grunty-pro@2x.png
+    image_styles: "max-width: 150px; margin-top: 18px"
+    features:
+      - name: Up to 10 Apps
+        description: An app is a single service that you want to migrate from Heroku to AWS. This plan allows you to migrate & run up to 10 apps. For more, please see the Enterprise plan.
+      - name: Single AWS Account
+        description: You can only migrate to a single AWS account with this plan. For multi-account setup, please refer to the Enterprise plan.
+      - name: Up to 10 Users
+        description: A user is a human or machine that accesses the Gruntwork code repos, training materials, or support services. For more than 10 users, see Gruntwork Enterprise.
+      - name: Gruntwork Pro Stack
+        description: You get access to the Gruntwork Pro Stack, which includes modules from the Gruntwork Lite Stack, and RDS, ElastiCache, Elasticsearch.
+      - name: Access to Docs
+        description: This plan gets you access to all the modules documentation and any production guides for free. For access to the DevOps Training Library, see the Enterprise plan.
+      - name: Community Support
+        description: Community Support is included with this plan. For faster response times, support via email, and private Slack chat, see the Enterprise plan.
+      - name: Contact us for estimated AWS costs*
```

Comment:
At this level, I think we should provide an estimate. We could use the AWS price estimator to figure out the rough cost of 3 envs, where each env has a VPC, NAT Gateway, 3 server cluster, RDS, etc... Let's say that works out to $500/mo. In this table, we could list "$500/month and up" and in the hover text explain pricing depends on the exact options the user picks (e.g., ECS or EKS or Fargate, EC2 instance types, RDS instance types, etc).

Reply:
>$500/month and up
Actually, I'm a big fan of the range option like you've suggested! Should have thought of that earlier, but glad it came up!

Reply:
The psychology behind this point was that migrating customers are used to only paying Heroku, whereas with our new plans that have to pay both us + AWS. We thought by adding a line here, it might make that fact more obvious. Agreed on adding the AWS price estimate.

Reply:
Yup, the psychology feels spot on to me!